SUPPORTED_OPS.rst

```diff
@@ -0,0 +1,72 @@
+Supported Operations
+####################
+
+.. table::
+
+    +------------------+-----------------------------------------+--------------+-----------------------+
+    | **Visibility**   | **Operation**                           | **Protocol** | **Number of Parties** |
+    +==================+=========================================+==============+=======================+
+    | Public & Private | Addition                                | - Falcon     |                    2+ |
```

Comment:
RSTensor supports addition and subtraction,the security gurantee breaks when we take it for the Falcon scenario , could we keep it as exactly 3 for add, sub in Falcon.

Reply:
Okey, no problem!!